src/components/Navigation.tsx

```diff
@@ -195,8 +311,70 @@ const Navigation: FC = () => {
                           </NavLink>
                         </li>
                       )}
+                      {enablePermissions && (
+                        <li className="p-side-navigation__item">
+                          <NavAccordion
+                            baseUrl="/ui/permissions"
+                            title={`Permissions`}
+                            iconName="user"
+                            label="Permissions"
+                            onToggle={adjustNavigationScrollForOverflow}
+                          >
+                            {[
+                              <li
+                                className="p-side-navigation__item"
+                                key="/ui/permissions/identities"
+                              >
+                                <NavLink
+                                  to="/ui/permissions/identities"
+                                  title="Identities"
+                                  onClick={softToggleMenu}
+                                  activeUrlMatches={["permissions/identity"]}
+                                  className="accordian-nav-secondary"
+                                >
+                                  Identities
+                                </NavLink>
+                              </li>,
+                              <li
+                                className="p-side-navigation__item"
+                                key="/ui/permissions/groups"
+                              >
+                                <NavLink
+                                  to="/ui/permissions/groups"
+                                  title="LXD groups"
+                                  onClick={softToggleMenu}
+                                  className="accordian-nav-secondary"
+                                >
+                                  Groups
+                                </NavLink>
+                              </li>,
+                              ...[
+                                enableIdpGroups ? (
```

Comment:
I think this should always be visible, better to show a warning in the idp group page in case this config key is not set.

Reply:
sure will go with this approach

Reply:
although currently we already have a card above the table, maybe we just display that if custom claim is set and hide it if not.

Reply:
actually I don't think we need to detect this for a warning. The warning should be there on the IDP groups page always to inform indirectly mapped groups will not show up in the UI for identities.

Reply:
There are various warnings at stake here as I see it
1. The intro text explaining what idp groups are
2. A warning if the config key `oidc.groups.claim` is empty.
3. A warning that the effective groups granted through the idp groups will not be visible on the identities in LXD.

Reply:
Will ask piper to have a look and see where we can slot these in, not sure if we should display them all on the same page as it would look quite weird. Maybe better way is to change the text in the same notification depending on the above criterias.

Reply:
Yes, agree, the warnings can all be in one place.

Though there is a chance for changing it, that additionally added copy will be lost, because the user has seen a warning in that  place before. But I think this doesn't apply, because in the lifecycle they would see all warnings initially and as things get configured the amount of warnings would be reduced.

Reply:
[Link](https://www.figma.com/file/0IsMmV1rsa6flOo6h4CjAV?type=design&node-id=1758-136992&mode=design#784779089) to question on figma.

Reply:
Spoke to @piperdeck , he agrees with the approach of changing text in the notification depending on the `oidc.groups.claim` setting.